include/cuco/detail/static_map/static_map_ref.inl

```diff
@@ -115,6 +115,26 @@ static_map_ref<Key, T, Scope, KeyEqual, ProbingScheme, StorageRef, Operators...>
   return impl_.capacity();
 }
 
+template <typename Key,
+          typename T,
+          cuda::thread_scope Scope,
+          typename KeyEqual,
+          typename ProbingScheme,
+          typename StorageRef,
+          typename... Operators>
+__host__ __device__ constexpr static_map_ref<Key,
+                                             T,
+                                             Scope,
+                                             KeyEqual,
+                                             ProbingScheme,
+                                             StorageRef,
+                                             Operators...>::extent_type
+static_map_ref<Key, T, Scope, KeyEqual, ProbingScheme, StorageRef, Operators...>::window_extent()
+  const noexcept
+{
+  return impl_.window_extent();
```

Comment:
This returns a `window_extent` which can't be implicitly converted to `size_type`. Do we want to return a `cuco::extent` instead? On second thought, maybe it's a better choice to be consistent with the return type. What do you think?

Reply:
Ah, right. Good point. Approved!